ui/admin-portal/src/environments/environment.staging.ts

```diff
@@ -29,5 +29,6 @@ export const environment = {
   s3BucketUrl: 'https://s3.us-east-1.amazonaws.com/files.tbbtalent.org',
   assetBaseUrl: '/admin-portal',
   environmentName: 'staging',
-  presetWorkspaceId: '987e2e02'
+  presetWorkspaceId: '987e2e02',
+  googleAnalyticsId: 'G-BPDYWB77Y3'
```

Comment:
Let's use the test ID for staging:
```suggestion
  googleAnalyticsId: 'G-K9ML1Y40B4' // for testing
```